docs/billing-integration.md

```diff
@@ -263,11 +266,43 @@ Additionally, if you've configured a `disk` endpoint, you can inspect the conten
 
 ### Getting a Reporting Secret for Testing
 
-There's currently no way to create a valid reporting secret for your application until it's published in GCP Marketplace (it doesn't need to be published publicly). You can use [this](https://storage.cloud.google.com/cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml) fake secret file which is structurally correct, but will fail authentication. Use the following procedure to get a valid secret for testing:
+#### Using a fake reporting secret
+
+We provide a [fake reporting secret](https://storage.cloud.google.com/cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml)
+which you can use for testing. This secret is structurally correct, but will fail authentication.
+
+You can use this secret directly using the following command:
+
+```shell
+mpdev install ... \
+  --parameters='{..., "reportingSecret": "gs://cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml"}'
+```
+
+Alternately, you can download and apply this secret to your target namespace and then point your application to it.
+
+```shell
+gsutil cp gs://cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml .
+
+# The downloaded secret doesn't have a name. Give it one and apply to the target namespace.
+
+echo "metadata: {name: fake-reporting-secret}" >> fake_reporting_secret.yaml
+kubectl apply -n my-namespace -f fake_reporting_secret.yaml
+
+# Now install the app. The value for reportingSecret matches the name we gave the fake secret.
+
+mpdev install ... \
+  --parameters='{..., "reportingSecret": "fake-reporting-secret"}'
+
+```
+
+#### Obtaining a real reporting secret
 
 * Once pricing has been submitted and published, submit and have published an initial version of your app. This version does not need to have working metering configuration, but should include the `REPORTING_SECRET` entry in schema.yaml.
```

Comment:
> Once pricing has been submitted and published, submit and have published an initial version of your app.
Can you fix that sentence? Should it be:
Once pricing has been submitted and published, submit and have publish an initial version of your app.